ocw_data_parser/ocw_data_parser.py

```diff
@@ -587,8 +588,9 @@ def extract_foreign_media_locally(self):
         os.makedirs(path_to_containing_folder, exist_ok=True)
         for media in self.large_media_links:
             file_name = media["link"].split("/")[-1]
+            response = requests.get(media["link"])
+            response.raise_for_status()
```

Comment:
Should the parsing be halted by an error if one of these links is no longer valid?  Maybe it should be in a try-catch block with exception logging [like below](https://github.com/mitodl/ocw-data-parser/pull/106/files#diff-893fe7ed068c91774e4c9132fefab09e62254d231c461f7debb7b6d70ce5ff59R793-R798)?

Reply:
Sounds good, I just made the change